CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
 
 ## Piwik 3.0.2
 
+### New Features
+* A new SMS provider for sms reports has been added: [ASPSMS.com](http://www.aspsms.com/en/?REF=227830)
+
+### Breaking Changes
+* SMS provider now need to define their credential fields by overwriting `getCredentialFields()`. This allows to have SMS providers that require more than only an API key.
```

Comment:
Is this still a breaking change? Or did we make it backwards compatible? Would be really great to keep BC and if not release 3.1. But ideally I would really want to not break BC in 3.X in general when it is possible to keep BC